ENIGMAsystem/SHELL/Graphics_Systems/None/fillin.cpp

```diff
@@ -154,7 +154,7 @@ namespace enigma_user
 	int draw_get_msaa_maxlevel(){return 0;}
 	bool draw_get_msaa_supported(){return false;}
 	void draw_set_msaa_enabled(bool enable){}
-	void draw_enable_alphablend(bool enable){}
+	void draw_set_blend(bool enable){}
```

Comment:
I think back in the day, I named these functions `draw_enable_foo` rather than `draw_set_foo_enabled`, but since it's the latter, now, we should keep consistent. Is there a reason to use this name in particular?